Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM ubuntu:bionic
+FROM ubuntu:jammy
 MAINTAINER patrick@oberdorf.net
 
 ENV VERSION 1.17.1
 
 WORKDIR /usr/local/src/
 ADD assets/sha256checksum sha256checksum
 
-RUN apt-get update && apt-get install -y \
+RUN apt update && apt install -y \
 	build-essential \
 	tar \
 	wget \
 	libssl-dev \
 	libevent-dev \
-	libevent-2.1-6 \
+	libevent-2.1-7 \
 	libexpat1-dev \
 	libexpat1 \
 	dnsutils \
-	&& wget http://www.unbound.net/downloads/unbound-${VERSION}.tar.gz -P /usr/local/src/ \
+	&& wget --no-hsts http://www.unbound.net/downloads/unbound-${VERSION}.tar.gz -P /usr/local/src/ \
```

Comment:
why --no-hsts? it works for me without disabling checking security headers. Can you remove this? I have tested everything else and will merge it right away :)

Reply:
I added this flag because we only do one download from that domain. Next time we build it becomes the first time again. Without the flag wget creates a `/root/.wget-hsts` file which will be added to the layer. Again this is only useful if you plan to visit the domain again. 

But if you still like me to remove this flag, I will.